careers/ladder.md

```diff
@@ -28,73 +28,183 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
 - Focuses on learning tools and processes.
+
+#### Impact
+
+- Can complete well-defined and subdivided tasks.
+
+#### Influence
+
 - Routinely pushes code through the entire lifecycle from development to production.
 - Continues to develop individual contributor skills, including source control, editors or test-driven development.
+- Understands team mission, goals and backlog.
+
+#### Discretion
+
+- Guided by objectives of project or ongoing assignments.
+- Plans own work priorities on a regular basis.
 
 ### IC2 Engineer
 
+#### tl;dr: is an effective team member
+
+#### Knowledge Leadership
+
 - Consistently writes and delivers correct and clean quality code with guidance.
+- Stewards a small-to-medium feature from technical design through completion.
+
+#### Impact
+
+- Makes steady, well-paced progress without the need for constant significant feedback from others.
+
+#### Influence
+
 - Self-sufficient and makes steady progress on tasks.
 - Knows when to ask for help and how to get unblocked.
-- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
-- Owns a small-to-medium feature from technical design through completion.
+- Effectively partners with other engineers and with team's cross-functional partners (including PM, Designer, Data Analyst).
 - Provides help and support outside of area under their responsibility.
 - Participates in the interviewing process of new team members.
 
+#### Discretion
+
+- Actively engages in team rituals.
+- Understands business impact of their work.
+
 ### IC3 Senior Engineer
 
-- Owns a service or large technology component.
-- Requires minimal direction or oversight, delivers quickly.
-- Understands elements of the business that their work supports.
+#### tl;dr: can drive change and impact at the team level
+
+#### Knowledge Leadership
+
 - Understands and makes well-reasoned design decisions and tradeoffs in their area.
 - Works in other areas of the codebase with guidance from owners of that system.
+- Takes responsibility for the maintenance and evolution of a significant service or technology.
+
+#### Impact
+
+- Delivers significant business value at team level.
+- Helps the team deliver on its overall goals.
+- Can drive specific initiatives.
+- Effectively translates ambiguous business problems into manageable tasks.
+
+#### Influence
+
 - Persistent in the face of roadblocks, dispatches them efficiently, pulling in others as necessary.
-- Has end-to-end responsibility on projects of increasing complexity and contributes to code that is shared across
-  projects.
+- Has end-to-end responsibility on projects of increasing complexity and contributes to code that is shared across projects.
+- Effectively contribute to longer-term product strategy.
 - Mentors junior engineers.
+
+#### Discretion
+
+- Requires minimal direction or oversight, delivers quickly.
+- Understands elements of the business that their work supports.
```

Comment:
Can you elaborate on the difference with “Understands business impact of their work.” in the IC2 Discretion section?

Reply:
Good question!
I interpret this as: "I understand how collections support merchandizing effort" vs "I understand merchandizing needs and therefore we should invest in collections", does that make sense?

Reply:
Absolutely 👍